assets/home/components/search-bar.jsx

```diff
@@ -11,7 +11,7 @@ export class SearchBar extends React.PureComponent {
                             <i className="icon--search icon--gray-light" />
                         </span>
                         <div className="search__form input-group searchForm--active">
-                            <form className="form-inline" action="/wire">
+                            <form className="form-inline" action="/wire" role="search" aria-label="search">
```

Comment:
there should be probably gettext used for the label

Reply:
ok let me fix it, tnx